recipe/meta.yaml

```diff
@@ -17,19 +17,17 @@ source:
   # lead-up until release -> don't use github sources on main
   # - url: https://github.com/apache/arrow/archive/refs/tags/apache-arrow-{{ version }}.tar.gz
   - url: https://dist.apache.org/repos/dist/release/arrow/arrow-{{ version }}/apache-arrow-{{ version }}.tar.gz
-    sha256: 5c70eafb1011f9d124bafb328afe54f62cc5b9280b7080e1e3d668f78c0e407e
+    sha256: 1304dedb41896008b89fe0738c71a95d9b81752efc77fa70f264cb1da15d9bc2
     patches:
-      # backport https://github.com/apache/arrow/pull/38375
-      - patches/0001-GH-38364-Python-Initialize-S3-on-first-use-38375.patch
       # workaround for https://github.com/apache/arrow/issues/37692
-      - patches/0002-fixture-teardown-should-not-fail-test.patch
+      - patches/0001-fixture-teardown-should-not-fail-test.patch
```

Comment:
No need to rename the patch manually; this is the format that `git format-patch` creates by default, but there's no inherent need that these patches be numbered sequentially.

Reply:
oh! ok, I also liked the consistency :)